#request
uint8 TRIANGLE = 0
uint8 SQUARE = 1
uint8 RECTANGLE = 2
uint8 CIRCLE = 3
uint8 HALFCIRCLE = 4

uint8 shape
geometry_msgs/Point shape_pos
geometry_msgs/Point[] shape_vertices
geometry_msgs/Point target_pos
---
#response

